internal/configuration/configuration.go

```diff
@@ -146,6 +168,30 @@ func (c *Data) GetPostgresConfigString() string {
 	)
 }
 
+func loadOSOClusters() (map[string]string, error) {
+	if _, err := os.Stat(defaultOsoClusterConfigPath); err != nil {
```

Comment:
ok, let's do this then from reading an ENV variable. Using the mounted secret seems a bit "dangerous" and a bit of an overkill.